src/platforms/f072/README.md

```diff
@@ -34,16 +34,21 @@ List the available devices
 dfu-util -l
 ```
 
-dfu-util should now list "[0483:df11]" and "@Internal Flash  /0x08000000/064*0002Kg".Compilethe firmware with:
+dfu-util should now list "[0483:df11]" and "@Internal Flash  /0x08000000/064*0002Kg".
 
+Compile the firmware with:
+
+```
+cd blackmagic
```

Comment:
This line looks like it's extra vs the other README's

Reply:
Yeah, this is because I mostly did a translation of the existing make commands to the corresponding meson ones, but I should've clean that up...

Reply:
Ah, yep, understood 🙂 - thank you!